.github/workflows/build.yaml

```diff
@@ -16,6 +16,7 @@ jobs:
         run: |
           sudo snap install yq
           sudo snap install rockcraft --classic --edge  --revision=687
+          sudo snap install charmcraft --classic --revision 1349
```

Comment:
Noob question: why different charmcraft here than in integration?

Reply:
AFAIK it is actually the same version, but in this case we install it while in case of integration test we update the installed one